model.py

```diff
@@ -122,3 +121,29 @@ def forward(self, inputs, order=False):
         # pred = self.classifier(outputs.hidden_states[:,-1])
         return outputs
 
+    def save(self, PATH: str, optimizer: torch.optim.Optimizer = None) -> None:
+        model_for_save = {
+            "SpecialLayer": self.SpecialLayer.state_dict(),
+            "prefix_encoder": self.prefix_encoder.state_dict(),
+            "classifier": self.GPI.classifier.state_dict(),
+        }
+        model_for_save.update(**self.config.__dict__)
+        model_for_save['text_encoder'] = model_for_save['text_encoder'].__dict__
```

Comment:
@SunCreation 
text_encoder and wav_encoder are pre-trained model, so is it necessary to save them?

Reply:
I think we can just give a model name(Klue/RoBERTa-large, whisper-base)? 
Please consider it. It is up to you!

Reply:
@YejinHwang909 
haha.. Actually, they are just Config class.
I also thought it's miss naming.😭